sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql

```diff
@@ -47,10 +47,14 @@ WITH combined_urlbar_picked AS (
       user_pref_browser_urlbar_show_search_suggestions_first AS BOOL
     ) AS show_search_suggestions_first,
     SAFE_CAST(user_pref_browser_urlbar_suggest_quicksuggest AS BOOL) AS suggest_quicksuggest,
+    SAFE_CAST(
+      user_pref_browser_urlbar_suggest_quicksuggest_nonsponsored AS BOOL
+    ) AS suggest_quicksuggest_nonsponsored,
     SAFE_CAST(
       user_pref_browser_urlbar_suggest_quicksuggest_sponsored AS BOOL
     ) AS suggest_quicksuggest_sponsored,
     user_pref_browser_urlbar_quicksuggest_onboarding_dialog_choice AS quicksuggest_onboarding_dialog_choice,
+    user_pref_browser_urlbar_quicksuggest_data_collection_enabled AS quicksuggest_data_collection_enabled,
```

Comment:
@rebecca-burwei Do you want to do anything special with the naming in urlbar_clients_daily here?

Reply:
no, personally, I prefer consistency in names across datasets over choosing a more meaningful, but different name in each derived table. better for automation in my downstream code.